polaris-react/src/components/Filters/Filters.module.css

```diff
@@ -217,22 +214,12 @@
 
 .ClearAll {
   margin-left: var(--p-space-200);
-
-  span {
-    font-size: var(--p-font-size-300);
```

Comment:
I agree. But I also think this is a more of a system issue with how our buttons function with micro + monochrome/plain. This is also the case where we have any plain or monochrome buttons inline with other non-plain buttons

Reply:
@heyjoethomas do you have any thoughts here? Particularly, should we make `<Button variant="monochromePlain | plain" size="micro">` have a `bodySm` text size?

Reply:
Alternatively, monochromePlain and plain could inherit the parent text size. This would help prevent areas where the parent text is not using a set text style:

<img width="296" alt="25-24-bwjx5-55c6p" src="https://github.com/Shopify/polaris/assets/11774595/a1815935-7e9b-4c93-b69c-decdbeeec580">

Reply:
@sam-b-rose Setting it to `bodySm` is probably safe given the intent here is to be small and compact. Otherwise it has the same text style as a default `Plan` button

Reply:
Updated the Button micro with plain/monochrome variant to use the `bodySm` style. This looks much better in the FiltersBar 💯